src/Microsoft.AspNetCore.Server.Kestrel.Configuration/KestrelServerOptionsSetup.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Certificates.Configuration;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Options.Infrastructure;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal
+{
+    internal class KestrelServerConfigureOptions : ConfigureDefaultOptions<KestrelServerOptions>
```

Comment:
Ok I'll file an issue to make them all pubternal post preview2 if I don't end up touching them again this week.  We need to standardize the naming then.

`ConfigureDefaultXyzOptions`?